src/network/graphql/Log.h

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#ifndef SRCNETGRAPHQLLOG_H
+#define SRCNETGRAPHQLLOG_H
+
```

Comment:
just a few days ago all guards were removed and #pragma once is used instead in all headers